tests/embedders/hex2vec/generation.py

```diff
@@ -0,0 +1,66 @@
+"""Test case generation for Hex2VecEmbedder."""
+from pathlib import Path
+from typing import Optional
+
+import geopandas as gpd
+from pytorch_lightning import seed_everything
+
+from srai.constants import WGS84_CRS
+from srai.embedders.hex2vec.embedder import Hex2VecEmbedder
+from srai.joiners import IntersectionJoiner
+from srai.loaders.osm_loaders import OSMPbfLoader
+from srai.loaders.osm_loaders.filters.osm_tags_type import osm_tags_type
+from srai.neighbourhoods import H3Neighbourhood
+from srai.regionizers import H3Regionizer
+from srai.utils import geocode_to_region_gdf
+from tests.embedders.hex2vec.constants import ENCODER_SIZES, TRAINER_KWARGS
+
+
+def generate_test_case(
```

Comment:
This function is not used at all. I understand that it can be used in the future to generate new use cases, but should we actually keep it?